storage/src/main/java/org/apache/kafka/storage/internals/log/CleanShutdownFileHandler.java

```diff
@@ -85,17 +83,16 @@ void write(long brokerEpoch, int version) throws Exception {
         }
     }
 
-    public long read() {
-        long brokerEpoch = -1L;
+    @SuppressWarnings("unchecked")
+    public OptionalLong read() {
         try {
             String text = Utils.readFileAsString(cleanShutdownFile.toPath().toString());
-            Map<String, String> content = new ObjectMapper().readValue(text, HashMap.class);
```

Comment:
ObjectMapper creation is expensive, it should be a static final field typically.

Reply:
Done.